src/lib/containers/download_list_v2/AvailableForDownloadTable.tsx

```diff
@@ -2,16 +2,12 @@ import React from 'react'
 import { QueryClient, QueryClientProvider } from 'react-query'
 import { useSynapseContext } from '../../utils/SynapseContext'
 import { SynapseErrorBoundary } from '../ErrorBanner'
-import DownloadListTable, {
-  DownloadListTableProps,
-} from './DownloadListTable'
+import DownloadListTable from './DownloadListTable'
 
 /**
  * Table of the files added to the Download Cart that are currently available for download.
  */
-export default function AvailableForDownloadTable(
-  props: DownloadListTableProps,
-) {
+export default function AvailableForDownloadTable() {
   const { accessToken } = useSynapseContext()
   const queryClient = new QueryClient({
```

Comment:
Unless you need the configured stale time, consider removing this QueryClient and the provider entirely to take advantage of having the cache shared outside of this component. We can also think about changing the default stale time.

Also, the staleTime comment is off by an order of magnitude 🙂